pkg/kustomize/kustomize.go

```diff
@@ -312,7 +314,25 @@ func KustomizationFor(
 		quayConfigTLSSources = append(quayConfigTLSSources, "ssl.key="+string(ctx.TLSKey))
 	}
 
+	// read the proxy configuration from the environment. makes sure we add the quay server
+	// host name to the list of addresses reachable without proxy (NO_PROXY).
+	proxyenv := httpproxy.FromEnvironment()
+	addrs := strings.Split(proxyenv.NoProxy, ",")
```

Comment:
note that if NoProxy is "", then addrs will be `[]string{""}`, not `[]string{}`.

Reply:
Addressed. PTAL.

Reply:
Are you happy with the proposed solution? If yes, can you please "resolve" this one?